src/object.h

```diff
@@ -237,8 +236,8 @@ struct object_kind {
 
 	/** Also saved in savefile **/
 
-	quark_t note_aware; 	/**< Autoinscription quark number */
-	quark_t note_unaware; 	/**< Autoinscription quark number */
+	char *note_aware; 	/**< Autoinscription */
+	char *note_unaware; 	/**< Autoinscription */
 
```

Comment:
With these changes, need logic in obj-init.c's cleanup_object() to release the memory allocated for note_aware and note_unaware.